### ISSUE TITLE
The scripts dont work

### ISSUE THREAD

**Issue by Tungee** [frederickk/scriptographer-scripts#1]
I downloaded your sciptographer scripts, but they gave only error messages on sc console. I tried to install your suggested folio.js but the most fail.
Best regards Tungee

**Comment by frederickk** [comment]
Yeah... this repo is pretty old and functions more as a warehouse for old code. For nearly 4 years I haven't had a version of illustrator that supports Scriptographer. It's worth noting that folio.js (in it's current state) is intended for the web as a library for paper.js only.

**Comment by Tungee** [comment]
Thanx Frederick for your fast response. I think im 4 years late to the party :)
Im really blown away by your scripts repo. The random toools are working well, they dont need the folio.js. I was particulary interested in the createGrid.js plugin, where i get this error message on console:
![bildschirmfoto 2017-01-12 um 13 50 41](https://cloud.githubusercontent.com/assets/3286171/21890487/37ae274a-d8ce-11e6-8eea-dd32f3a76668.png)
Best Regards Hakan

**Comment by frederickk** [comment]
ahhh. let me take a look at that particular script. perhaps there are some simple modifications I could make. no promises though :)

**Comment by Tungee** [comment]
Ken, das wäre verdammt geil!

LG Tungee

2017-01-13 7:56 GMT+01:00 kenfrederick <notifications@github.com>:

**Comment by frederickk** [comment]
Ok, I believe I've found the issue.

Line 16 of "createGrid_0_0.js" is the following:
```
include('libraries/folio/scriptographer.folio.js');
```

So you're likely missing "scriptographer.folio.js" You can download the library [here](https://github.com/frederickk/scriptographer-scripts/files/737347/scriptographer.folio.zip) and simply update Line 16 to the path to wherever you save "scriptographer.folio.js".

Viel Glück!
